application/F3DOptionsParser.cxx

```diff
@@ -289,11 +289,10 @@ void ConfigurationOptions::GetOptions(F3DAppOptions& appOptions, f3d::options& o
   try
   {
     cxxopts::Options cxxOptions(this->ExecutableName, F3D::AppTitle);
-    cxxOptions.positional_help("file1 file2 ...");
+    cxxOptions.custom_help("[OPTIONS...] file1 file2 ...");
 
     // clang-format off
     auto grp0 = cxxOptions.add_options("Applicative");
-    this->DeclareOption(grp0, "input", "", "Input files", inputs, LocalHasDefaultNo, MayHaveConfig::YES , "<files>");
```

Comment:
that removes the `--input` option. this would require deprecation

Reply:
We can, but we need to deprecate it first. So we need to keep supporting it somehow, even if it doesnt support commas.